scripts/build.py

```diff
@@ -1190,6 +1206,8 @@ def compile_typescript_files(project_dir):
         project_dir: str. The project directory which contains the ts files
             to be compiled.
     """
+    test_compiled_js_dir_is_valid()
+    safe_delete_directory_tree(COMPILED_JS_DIR)
     print 'Compiling ts files...'
     cmd = ['./node_modules/typescript/bin/tsc', '--project', project_dir]
     subprocess.check_call(cmd)
```

Comment:
> Yes, I mean to say that when typescript finishes initial compilation we can return from this function since the watcher is running in the background and the initial set of compiled files is also produced.

Are you planning to change the docstring/comments to clarify?

Reply:
The comment [here](url) mentions the point that `whenever typescript does the compilation, the last line reads Watching for file changes`. So, the update required here is to change compilation -> initial compilation?

Reply:
Yes, please make that change. Thanks!

Reply:
Done.